### ISSUE TITLE
[VO] unicode "Minus Sign" not read when used in aria-valuetext

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#37]
Unicode values are read inconsistently when used with aria-valuetext.
JSFiddle that isolates this problem: https://jsfiddle.net/s8m2spxc/3/

This was investigated and found in https://github.com/phetsims/john-travoltage/issues/238

Part of the problem is that different VoiceOver voices read things differently.

Testing with voice "Karen" -
>When focus lands on the first slider (using '\u2212' - "Minus Sign"), "Karen" reads the "Minus sign". When changing the value on that slider, the minus sign is never read.

>When focus lands on the second slider (using '\u002D' - "hyphen-minus"), "Karen" does not read "Minus Sign", but when the value changes, the minus sign is read.

Testing with voice "Alex"
>We have the exact same behavior as above, but when changing the value with '\u2212' - "Minus Sign" in the aria-valuetext, the unicode value "Minus Sign" is read.

>Using the default voice works fine but any other voice doesn't read unicode "Minus Sign" when used in aria-valuetext.

**Comment by jessegreenberg** [comment]
Verified this is still a problem and submitted a but report to Apple.

**Comment by jessegreenberg** [comment]
Update 10/3/17, no response from Apple.

**Comment by jessegreenberg** [comment]
Update 1/8/18, no response from Apple.
